docs/usage/project_namespace_access.md

```diff
@@ -11,9 +11,9 @@ kubectl -n project-abc create sa robot-user
 ```
 
 ### Request a token for a Service Account
-A token for the "robot-user" `ServiceAccount` can be requested via the [TokenRequest API](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/).
+A token for the "robot-user" `ServiceAccount` can be requested via the [TokenRequest API](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/) in several ways:
 
-The request can be made with `kubectl`
+- using `kubectl`
```

Comment:
I thought of this also, but it was somehow strange to me since kubectl >= 1.24 does not exclude this type of calling the tokenrequest API. But I guess it is OK since it is easier to use the `create token` command if kubectl >= v1.24. This is also why I put this option first.